src/test/lib/statistics/cardinality_estimator_test.cpp

```diff
@@ -1087,4 +1087,25 @@ TEST_F(CardinalityEstimatorTest, WindowNode) {
   EXPECT_TRUE(result_table_statistics->column_statistics.at(2));
 }
 
+TEST_F(CardinalityEstimatorTest, StatisticsCaching) {
+  // Enable statistics caching.
```

Comment:
Difference to `TEST_F(CostEstimatorLogicalTest, CardinalityCaching)`?

Reply:
Same. The test case for the cost estimator is just around to ensure the caching flag is correctly passed to the cardinality estimator.